reconcile/templating/lib/rendering.py

```diff
@@ -78,7 +78,7 @@ def render_output(self) -> str:
         pass
 
     def render_target_path(self) -> str:
-        return self._render_template(self.template.target_path)
+        return self._render_template(self.template.target_path).strip().strip(r"\n")
```

Comment:
don't you need an other `.strip()` at the end if new line and spaces are mixed?

Reply:
we need to leave one newline at EOF, will this strip everything?

Reply:
We want to strip anything because it's a file path string.

Reply:
I think `\n` is also removed in `strip()`, any edge cases?

Reply:
jupp, you're right, one `strip()` is enough. fixed in [3a2ec52](https://github.com/app-sre/qontract-reconcile/pull/4499/commits/3a2ec520b8e97e74ddf01189d537552c9a323610)